src/core_plugins/vega_vis/public/vega_vis_editor.template.html

```diff
@@ -0,0 +1,47 @@
+<!--TODO: add collapsible-sidebar class (doesn't work in 5.5 with the split.js) -->
+<div class="sidebar-container vegaEditor" style="margin-right: 0;">
+  <div class="vegaEditorToolbar">
+    <div>Reformat&nbsp;as:</div>
+    <button
+      ng-click="formatHJson()"
+      tooltip="Format graph code as HJSON"
+      class="kuiButton kuiButton--basic"
+    >
+      <span>HJSON</span>
```

Comment:
I'd remove the HJSON support. I think we should keep this as small and functional as possible. The problem with bringing in yet another flavor of language, is similar to what we're experiencing with our flavors of markdown-support. Just more 3rd party dependencies and all the slight mismatches and long-term issues that arise with that.

I think the benefits over the long run (a leaner Kibana), will outweigh the benefits in the short run (slightly better user experience). This is a first introduction of this plugin, I'd start with the smallest thing that works and build up from there. It's better to be able to dial in features, than have to prune features.

Reply:
If we remove HJSON support, it would be impossible to have comments in JSON - which we heavily use in all of the examples including the first one. JSON is painful for humans to write in...  and YAML has nasty automagical things like references and `yes|no` is actually a `true|false`.  Do we want to strip all the comment capabilities?

Reply:
I'd be OK with stripping the comment capabilities.

@timroes @ppisljar  what do you think?